content/en/blog/2024/kubecon-eu.md

```diff
@@ -0,0 +1,116 @@
+  by Adriana Villela, ServiceNow Cloud Observability (formerly Lightstep) &
+  Reese Lee, New Relic<br> Tuesday, March 19th • 10:00 - 10:25
+- **[How to Think About Instrumentation Overhead](https://sched.co/1YFfb)**<br>
+  by Jason Plumb, Splunk<br> Tuesday, March 19th • 11:05 - 11:30
+- **[TTChat’s Story: Connect Metrics, Logs and Traces with eBPF](https://sched.co/1YFfe)**<br>
+  by Zhu Jiekun, Quwan<br> Tuesday, March 19th • 11:05 - 11:30
+- **[Panel: OpenTelemetry: Realizing the Value of Open Standards](https://sched.co/1YFgW)**<br>
+  by Daniel Gomez Blanco, Skyscanner; Marcin Sodkiewicz, Ryanair; Iris Dyrmishi,
+  Miro; Hope Oluwalolope, Microsoft<br> Tuesday, March 19th • 12:15 - 12:50
+- **[Telemetry Showdown: Fluent Bit Vs. OpenTelemetry Collector - a Comprehensive Benchmark Analysis](https://sched.co/1YFhI)**<br>
+  by Henrik Rexed, Dynatrace<br> Tuesday, March 19th • 13:30 - 13:55
+- **[Monitoring Serverless Workloads with OpenTelemetry and Prometheus](https://sched.co/1YFhh)**<br>
+  by Ridwan Sharif, Google<br> Tuesday, March 19th • 14:05 - 14:30
+- **[Observability at the Edge: Instrumenting WebAssembly with OpenTelemetry](https://sched.co/1YFik)**<br>
+  by Dan Norris & Joonas Bergius, Cosmonic<br> Tuesday, March 19th • 15:15 -
+  15:40
+- **[Real-World Sampling – Lessons Learned After Reducing ~80% of Our O11y Costs](https://sched.co/1YFii)**<br>
+  by Juraci Paixão Kröhling, Grafana Labs & Alexandre Magno Prado Machado, Pismo
+  Soluções Tecnológicas<br> Tuesday, March 19th • 15:15 - 15:40
+- **[⚡ Lightning Talk: Not Just Enterprise. Modern Java App CI/CD Observability with OTel, Quarkus and Gradle](https://sched.co/1YFin)**<br>
+  by Oleg Nenashev, WireMock<br> Tuesday, March 19th • 15:45 - 15:50
+- **[Shift Into an Observability Mindset with OpenTelemetry](https://sched.co/1YFjB)**<br>
+  by Daniel Gomez Blanco, Skyscanner<br> Tuesday, March 19th • 15:45 - 16:15
+- **[⚡ Lightning Talk: Federated Search Over Distributed Observability Data](https://sched.co/1YFjC)**<br>
+  by Kalyan Kolachala, Intuit<br> Tuesday, March 19th • 15:55 - 16:00
+- **[⚡ Lightning Talk: Application Security Through the Lens of OpenTelemetry - Yosef Arbiv, Outshift by Cisco](https://sched.co/1YFf5)**<br>
+  by Kalyan Kolachala, Intuit<br> Tuesday, March 19th • 16:05 - 16:10
+- **[Lazy Robots: Telemetry Buffering on Android](https://sched.co/1YFk3)**<br>
+  by Cesar Munoz, Elastic & Jason Plumb, Splunk<br> Tuesday, March 19th •
+  17:00 - 17:25
+- **[OpAMP in Action: User Configurable Observability Pipelines](https://sched.co/1YFk6)**<br>
+  by Srikanth Chekuri, SigNoz<br> Tuesday, March 19th • 17:00 - 17:25
+
+{{% alert title="Important access note" color="danger" %}}
+
+You need an _in-person all-access_ pass for on-site access to **Observability
+Day**. For details, see [KubeCon registration][]. If you have a virtual ticket,
+you will be able to follow **Observability Day** through a live stream.
+
+{{% /alert %}}
+
+## OpenTelemetry Observatory
```

Comment:
I know we're still shaping up the feedback sessions at the observatory, but is this something we want to mention here @open-telemetry/end-user-wg ?

Reply:
@danielgblanco most definitely - I added some suggestions around that